src/sagemaker/fw_utils.py

```diff
@@ -59,8 +59,8 @@
     "local_gpu",
 )
 SM_DATAPARALLEL_SUPPORTED_FRAMEWORK_VERSIONS = {
-    "tensorflow": ["2.3.1", "2.3.2", "2.4.1"],
-    "pytorch": ["1.6.0", "1.7.1", "1.8.0"],
+    "tensorflow": ["2.3.1", "2.3.2", "2.4.1", "2.4", "2.3"],
```

Comment:
can we do ["2.3", "2.3.1", "2.3.2", "2.4", "2.4.1"] for the version consistency?

Reply:
sure